airbyte-integrations/connectors/source-harvest/integration_tests/expected_records.jsonl

```diff
@@ -102,12 +102,12 @@
-{"stream":"time_team","data":{"user_id":3758382,"user_name":"[SAMPLE] Hiromi Hourglass","is_contractor":false,"total_hours":162.23,"billable_hours":129.77,"currency":"USD","billable_amount":11227.5,"from":"20210101","to":"20220101"},"emitted_at":1672846398446}
-{"stream":"time_team","data":{"user_id":3758381,"user_name":"[SAMPLE] Kiran Kronological","is_contractor":false,"total_hours":156.68,"billable_hours":131.12,"currency":"USD","billable_amount":11528.75,"from":"20210101","to":"20220101"},"emitted_at":1672846398447}
-{"stream":"time_team","data":{"user_id":3758383,"user_name":"[SAMPLE] Tamara Timekeeper","is_contractor":false,"total_hours":154.59,"billable_hours":76.22,"currency":"USD","billable_amount":9329.25,"from":"20210101","to":"20220101"},"emitted_at":1672846398447}
-{"stream":"time_team","data":{"user_id":3758384,"user_name":"[SAMPLE] Warrin Wristwatch","is_contractor":false,"total_hours":160.07,"billable_hours":78.68,"currency":"USD","billable_amount":8807.75,"from":"20210101","to":"20220101"},"emitted_at":1672846398447}
-{"stream":"time_team","data":{"user_id":3758380,"user_name":"Airbyte Developer","is_contractor":false,"total_hours":8.01,"billable_hours":8.01,"currency":"USD","billable_amount":80.1,"from":"20210101","to":"20220101"},"emitted_at":1672846398448}
 {"stream":"project_budget","data":{"project_id":28671446,"project_name":"Fixed Fee Project","client_id":10748670,"client_name":"[SAMPLE] Client A","budget_is_monthly":false,"budget_by":"project_cost","is_active":true,"budget":21900,"budget_spent":19164.5,"budget_remaining":2735.5},"emitted_at":1672846398699}
 {"stream":"project_budget","data":{"project_id":28671449,"project_name":"Non-Billable Project","client_id":10748670,"client_name":"[SAMPLE] Client A","budget_is_monthly":false,"budget_by":"project","is_active":true,"budget":160,"budget_spent":140.97,"budget_remaining":19.03},"emitted_at":1672846398700}
 {"stream":"project_budget","data":{"project_id":28671448,"project_name":"Monthly Retainer","client_id":10748671,"client_name":"[SAMPLE] Client B","budget_is_monthly":true,"budget_by":"project_cost","is_active":true,"budget":21910,"budget_spent":0,"budget_remaining":21910},"emitted_at":1672846398700}
 {"stream":"project_budget","data":{"project_id":28671447,"project_name":"Time & Materials Project","client_id":10748671,"client_name":"[SAMPLE] Client B","budget_is_monthly":false,"budget_by":"person","is_active":true,"budget":150,"budget_spent":166.21,"budget_remaining":-16.21},"emitted_at":1672846398700}
+{"stream":"time_team","data":{"user_id":3758382,"user_name":"[SAMPLE] Hiromi Hourglass","is_contractor":false,"weekly_capacity":144000,"avatar_url":"https://cache.harvestapp.com/assets/avatars/sample_avatar_2.png","total_hours":162.23,"billable_hours":129.77,"currency":"USD","billable_amount":11227.5,"from":"20210101","to":"20220101"},"emitted_at":1675795652626}
```

Comment:
Why did these change? Looks like harvest wasn't failing nightlies, and I wouldn't expect this change to affect the expected records

Reply:
They constantly change
if @darynaishchenko last PR https://github.com/airbytehq/airbyte/pull/22406 works ok
I will keep her last version